app/workers/search/classified_listing_es_index_worker.rb

```diff
@@ -0,0 +1,12 @@
+module Search
+  class ClassifiedListingEsIndexWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority
+
+    def perform(classified_listing_id)
+      classified_listing = ::ClassifiedListing.find_by!(id: classified_listing_id)
```

Comment:
I think we can just use `find` here, it even has a slightly better exception message:

```
ActiveRecord::RecordNotFound: Couldn't find ClassifiedListing with 'id'=X
```

vs 

```
ActiveRecord::RecordNotFound: Couldn't find ClassifiedListing
```

Reply:
Good idea - updated!